frontend/src/v5/validation/shared/validators.ts

```diff
@@ -20,6 +20,8 @@ import * as Yup from 'yup';
 
 export const trimmedString = Yup.string().transform((value) => value.trim());
 
+export const trimmedLowercaseString = trimmedString.transform((value) => value.toLocaleLowerCase());
```

Comment:
This means that project names are now all lowercase. Is this what we want?

Reply:
This change also means that the project settings behaves strangely.
If you change the name to be the same but with a different case then you get a success message although nothing actually changes
![image](https://user-images.githubusercontent.com/62481250/230405465-493e3f8e-b1f9-4a6a-9a89-94a2c0ca8861.png)
(image shows trying to rename 'all cap' -> 'ALL CAP'. The project name continues being 'all cap')